Documentation/index.md

```diff
@@ -8,18 +8,18 @@ For soldering irons that are designed to be powered by batteries (TS100 & Pineci
 
 Currently **31** languages are supported. When downloading the firmware for your soldering iron, take note of the language code in the file name.
 
-This project is considered feature complete for use as a soldering iron, _so please suggest any feature improvements you would like!_
+This project is considered stable & feature complete for everyday use with a supported device, _so please suggest any feature improvements you would like!_
 
 _This firmware does **NOT** support the USB port while running for changing settings. This is done through the onscreen menu only. Logos are edited on a computer and flashed like firmware._
 
 |   Device   | DC  | QC  | PD  | EPR | BLE | Battery | Recommended |
 | :--------: | :-: | :-: | :-: | :-: | :-: | :-----: | :---------: |
-|   MHP30    | ❌  | ❌  | ✔️  | ❌  | ❌  |   ❌    |     ✔️      |
-| Pinecil V1 | ✔️  | ✔️  | ✔️  | ❌  | ❌  |   ✔️    |     ✔️      |
-| Pinecil V2 | ✔️  | ✔️  | ✔️  | ✔️  | ✔️  |   ✔️    |     ✔️      |
-|   TS80P    | ❌  | ✔️  | ✔️  | ❌  | ❌  |   ✔️    |     ✔️      |
-|   TS100    | ✔️  | ❌  | ❌  | ❌  | ❌  |   ✔️    |     ❌      |
-|    TS80    | ❌  | ✔️  | ❌  | ❌  | ❌  |   ✔️    |     ❌      |
+|   MHP30    | ❌  | ❌  | ✔️   | ❌  | ❌  |   ❌    |     ✔️       |
+| Pinecil V1 | ✔️   | ✔️   | ✔️   | ❌  | ❌  |   ✔️     |     ✔️       |
```

Comment:
Looks like you have broken the vertical alignment in the text form here which is _nice_ to have :)

Reply:
Yea... it seems different editors/md viewers/IDEs align tables with UTF chars in a _slightly_ different way. Sorry for that, of course I will fix it.

Reply:
Right... so I just copy-pasted back _unaligned-for-me_ table from original `index.md` from `dev` and now it even doesn't show the diff. ;)
Ok, note to myself : do not mess with md tables with utf symbols inside. :D

Reply:
Yeah quite surprised, most editors I've used have handled it fine so far 😓